src/app.ts

```diff
@@ -430,9 +432,13 @@ export function newApp({
             serviceProviderConfig
           }).run(),
         err => new Error(`Unexpected error initizing Spid Login: [${err}]`)
-      );
+      ).map(withSpidApp => ({
+        ...withSpidApp,
+        spidConfigTime: timer.getElapsedMilliseconds()
```

Comment:
@BurnedMarshal @gunzip Sorry guys, I didn't mean to mess with this pull request. Still, I don't understand what's the point here. 

I realised that my feedbacks might have been somehow misleading.